base/abstractarraymath.jl

```diff
@@ -92,6 +92,10 @@ imag{T<:Real}(x::AbstractArray{T}) = zero(x)
 +{T<:Number}(x::AbstractArray{T}) = x
 *{T<:Number}(x::AbstractArray{T,2}) = x
 
+function lerpi(j::Integer, d::Integer, A::AbstractArray, B::AbstractArray)
```

Comment:
I meant replacing the call sites with `lerpi.()` anywhere it needs to be called on arrays, instead of defining this method at all